admin/test_langs.php

```diff
@@ -225,6 +225,8 @@ protected function checkTokens() {
 			return false;
 		}
 
+		$t_do_html_check = extension_loaded('dom') && extension_loaded( 'libxml' );
```

Comment:
In case of non-loaded extensions, some addional line with a warning like `Syntax check for HTML tags can't be executed due to missing "dom" or "libxml" PHP extensions` would be good.

Reply:
Good idea, I'll add that.